src/DssSpell.t.sol

```diff
@@ -1122,12 +1111,12 @@ contract DssSpellTest is DSTest, DSMath {
 
         // Deposit collateral, generate DAI
         assertEq(vat.dai(address(this)), 0);
-        vat.frob(_ilk, address(this), address(this), address(this), int(amount), int(dust * WAD));
+        vat.frob(_ilk, address(this), address(this), address(this), int(amount), int(dust));
```

Comment:
The only comment I'd like to give is regarding the semantics of using `dust` in this case. I guess it wants to be proved that the minimum amount can be drawn. However `dust` should be converted via `rate` in order to actually have the corresponding `dart` value. It's true that we are not calling `drip` before here so `dai amt == dart` in this specific case, but we might want to express it in the most general case.